### PR TITLE
Trev8939/sceneLayer

### PR DESCRIPTION
**Pull request by TADraeseke**
Adds a sample demonstrating use of a `SceneLayer` in a `SceneView`

https://devtopia.esri.com/runtime/java-android-sdk/issues/1322

### CODE REVIEW

**Comment by TADraeseke** [comment]
https://devtopia.esri.com/runtime/java-android-sdk/issues/1322

File: scene-layer/build.gradle [hunk 1]
```diff
@@ -0,0 +1,24 @@
+apply plugin: 'com.android.application'
+
+android {
+    compileSdkVersion rootProject.ext.compileSdkVersion
+    buildToolsVersion rootProject.ext.buildToolsVersion
+
+    defaultConfig {
+        applicationId "com.esri.arcgisruntime.sample.displayscene"
```
Comment (doneill) [inline scene-layer/build.gradle]: update `applicationId` to be specific to this sample, e.g. `com.esri.arcgisruntime.sample.scenelayer`.

**Comment by TADraeseke** [comment]
Thanks @doneill! Comment addressed.

**Comment by doneill** [comment]
@TADraeseke Looks good to merge!

**Pull request merged**